Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/livy/interactive/Session.java

```diff
@@ -52,7 +54,7 @@
 import static rx.exceptions.Exceptions.propagate;
 
 public abstract class Session implements AutoCloseable, Closeable {
-    private static final String REST_SEGMENT_SESSION = "sessions";
+    private static final String REST_SEGMENT_SESSION = "/sessions";
```

Comment:
The `sessions` isn't fixed as the root segment

Reply:
Done.